recipe/meta.yaml

```
@@ -28,8 +30,9 @@ outputs:
       script: python -m pip install . --no-deps --ignore-installed -vv
     requirements:
       host:
+        - __unix
+        - python >=3.7
         - pip
-        - python
       run:
         - python
```

I think this is needed in every `requirements > X` section (that was what was done in https://github.com/conda-forge/cylc-flow-feedstock/pull/33)
```suggestion
        - __unix
        - python >=3.7
```